### PR TITLE
[ADD] added microsite creation screens

### PR DESCRIPTION
**Pull request by ekediala**
Added first two screens for creating a microsite

### CODE REVIEW

File: src/components/screens/onboardingScreens/WelcomeScreen.jsx [hunk 1]
```diff
@@ -35,7 +39,7 @@ const WelcomeScreen = () => {
     {
       type: TYPE_PRIMARY,
       text: SIGN_UP,
-      onPress: () => NavigationService.navigate(SIGN_UP_ROUTE)
+      onPress: () => NavigationService.navigate(CREATE_MICROSITE_ROUTE)
```
Comment (mrayor) [inline src/components/screens/onboardingScreens/WelcomeScreen.jsx]: Fix this back to sign up route.

Comment (ekediala) [reply src/components/screens/onboardingScreens/WelcomeScreen.jsx]: Done

File: src/components/createMicrosite/reducer.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    loading: false,
+    tags: [],
+};
+
+const micrositeReducer = (state = initialState, action) => {
+    switch (action.type) {
+    case UPDATE_MICROSTORE:
+        return Object.assign({}, state, action.payload);
+    case SET_TAGS:
+        return { ...state, tags: action.payload };
```
Comment (parkerthegeniuschild) [inline src/components/createMicrosite/reducer.js]: Why are you not consistent?

Comment (ekediala) [reply src/components/createMicrosite/reducer.js]: For update, I do not know what keys are there at each time as they are being sent from different screens so I am just dumping them in. I could extract it into a variable and spread it in but I just felt it wasn't worth another line of code.

**Comment by parkerthegeniuschild** [comment]
Okay cool, so I just ran this branch on expo and I could not find the microsite form or anything relating to microsite . .  Please come help me out . .

File: src/components/screens/onboardingScreens/WelcomeScreen.jsx (hunk 1 above)
Comment (mrayor) [reply src/components/screens/onboardingScreens/WelcomeScreen.jsx]: I mean this file @parkerthegeniuschild

**Comment by parkerthegeniuschild** [comment]
> @parkerthegeniuschild , check this file....temporarily do this to check the UI.

which file ???

**Pull request merged**